autopts/wid/gap.py

```diff
@@ -1544,6 +1551,60 @@ def hdl_wid_309(_: WIDParams):
     return stack.gap.wait_periodic_transfer_received(10)
 
 
+def hdl_wid_350(_: WIDParams):
+    # Please synchronize with Broadcast ISO request.
+    return True
+
+
+def hdl_wid_351(_: WIDParams):
+    # Wait for Broadcast ISO request.
+    stack = get_stack()
+    return stack.gap.gap_wait_for_iso_channel_status(types.IsoChannelStatus.CONNECTED)
+
+
+def hdl_wid_352(_: WIDParams):
+    # Please click OK when IUT establishes BIG sync, and ready to receive ISO data.
+    stack = get_stack()
+    btp.gap_iso_sync_big()
+    is_con = stack.gap.gap_wait_for_iso_channel_status(types.IsoChannelStatus.CONNECTED)
+    return is_con
+
+
+def hdl_wid_353(_: WIDParams):
+    # Please configure IUT security to Mode 3 Level 2 or Level 3.
+    return True
+
+
+def hdl_wid_354(_: WIDParams):
+    # Please synchronize to lower tester in Security Mode 3 Level 2 or 3.
+    # If IUT failed to synchnorize to the lower tester, please click ok.
+    btp.gap_iso_sync_big()
+
+    stack = get_stack()
+    is_discon = stack.gap.gap_wait_for_iso_channel_status(types.IsoChannelStatus.DISCONNECTED)
+    return is_discon
+
+
+def hdl_wid_356(_: WIDParams):
+    # Please broadcast valid ISO data packets (more than 3 packets).
```

Comment:
It is Ok as it is, but I wonder if we can retrieve min number of packets required to send from description and use it in code?